packages/ember/index.js

```diff
@@ -336,9 +334,7 @@ Object.defineProperty(Ember.run, 'currentRunLoop', {
 
 // ****@ember/-internals/metal****
 
-// Using _globalsComputed here so that mutating the function is only available
-// in globals builds
-const computed = metal._globalsComputed;
+const computed = metal.computed;
```

Comment:
Same thing here, the property access on these functions should not be available if you are importing the modules directly.

Reply:
Ah, I see, that makes sense. I'll add it back here, though it does make these tests a bit trickier...